File: api.unitethiscity.com/Models/SummaryAnalytics.cs
/******************************************************************************
 * Filename: SummaryAnalytics.cs
 * Project:  UTC WebAPI
 * 
 * Description:
 * POCO for representing model of summary analytics
******************************************************************************/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace com.unitethiscity.api.Models
{
    public class SummaryAnalytics
    {
        public string Name { get; set; }
        public int Total { get; set; }
        public double Percent { get; set; }
        public int Group { get; set; }
    }
}

File: android/unitethiscity/uniteThisCity/src/main/java/com/unitethiscity/data/SummaryAnalytics.java
package com.unitethiscity.data;

/**
 * Created by Don on 2/20/2016.
 */
public class SummaryAnalytics {

    private String Name;
    private int Total;
    private double Percent;
    private int Group;

    public SummaryAnalytics(String name, int total, double percent, int group) {
        Name = name;
        Total = total;
        Percent = percent;
        Group = group;
    }

    public String getName() {
        return Name;
    }

    public int getTotal() {
        return Total;
    }

    public double getPercent() {
        return Percent;
    }

    public int getGroup() {
        return Group;
    }

}
